### DISCUSSION TITLE
Numeric Habits: Reached target should effect following days

### DISCUSSION THREAD

**Discussion by Mr-Tin-25** [iSoron/uhabits#1168] [Feature Requests] [answered]
Hey,

right know:
When I set a numeric target to 10 units every week, this 10 units will be split up to 1.45 units every day. If I do this 10 units at one day, it doesn't have an effect at the following days. Every day 1.45 units are expected.

It would be great, if it would have at the following days and that there would be a check with this units per weak were reached.

Thanks

**Comment by baincd** [comment, marked as answer]
I would really like this as well!

I think this may be a duplicate of #1180
